modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -22,23 +22,9 @@ import '../css/main.scss';
 const BalloonEditor = ({config = {}, contents, name, ...otherProps}) => {
 	const [cssClass, setCssClass] = useState('');
 
-	const defaultExtraPlugins = 'balloontoolbar,floatingspace';
-
-	const getConfig = () => {
-		const extraPlugins = config.extraPlugins
-			? `${config.extraPlugins}`
-			: '';
-
-		return {
-			...config,
-			extraAllowedContent: '*',
-			extraPlugins: `${extraPlugins}${defaultExtraPlugins}`,
-		};
-	};
-
 	return (
 		<Editor
-			config={getConfig()}
+			config={config}
```

Comment:
Hey @ambrinchaudhary I think this is a good start :+1: , but if I remember correctly; yesterday we spoke about making sure we'd always get the required config even from a JS Widget only application.

I'm not saying we should do it right now in order to merge this, but I think it's something we should keep in mind and have a thought about how we'd achieve that. What do you think?